private-templates-service/adaptivecards-templating-service/src/storageproviders/MongoDBProvider.ts

```diff
@@ -86,6 +86,7 @@ export class MongoDBProvider implements StorageProvider {
   // Updates Only one user
   async updateUser(query: Partial<IUser>, updateQuery: Partial<IUser>): Promise<JSONResponse<Number>> {
     let userQuery: any = this._constructUserQuery(query);
+    updateQuery = MongoUtils.removeUndefinedFields(updateQuery);
```

Comment:
This change as well as the one below is to address results of the fields updated that are not defined in the updateQuery. They were set to null by Mongo.

Reply:
Thanks Seva.